bin/zkServer.sh

```diff
@@ -37,73 +37,62 @@ fi
 # up the JVM to accept JMX remote management:
 # http://java.sun.com/javase/6/docs/technotes/guides/management/agent.html
 # by default we allow local JMX connections
-if [ "x$JMXLOCALONLY" = "x" ]
-then
-    JMXLOCALONLY=false
+if [ -n "$JMXLOCALONLY" ]; then
+  JMXLOCALONLY=false
```

Comment:
Shouldn't this be:

```suggestion
if [ -z "$JMXLOCALONLY" ]; then
  JMXLOCALONLY=false
```

Reply:
Yeah, it got me pretty messed up the x=x xDD. You are absolutely right, I'm gonna change it 👍 
Thanks!